delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/ViewRef.scala

```diff
@@ -0,0 +1,12 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.model
+
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+
+/**
+  * View reference.
+  *
+  * @param project  the project to which the view belongs
+  * @param viewId   the view id
+  */
+final case class ViewRef(project: ProjectRef, viewId: Iri)
```

Comment:
Since we have now the sdk-views module, we should put this `ViewRef` there, instead of having it in each view related module